app/code/Magento/SalesRule/Model/ResourceModel/Coupon/Usage.php

```diff
@@ -44,13 +44,20 @@ public function updateCustomerCouponTimesUsed($customerId, $couponId, $increment
         );
 
         $timesUsed = $connection->fetchOne($select, [':coupon_id' => $couponId, ':customer_id' => $customerId]);
-
         if ($timesUsed !== false) {
+            $updatedValue    = $increment ? 1 : -1;
```

Comment:
Please, remove extra spaces

Reply:
Hi @sidolov,
 I have removed the extra spaces. Kindly check and let me know.